lib/localeapp/cli/pull.rb

```diff
@@ -26,6 +29,17 @@ def report_failure(response)
         @output.puts "Failed!"
         fail APIResponseError, "API returned #{response.code} status code"
       end
+
+      private
+
+      def yaml_data(response)
```

Comment:
I think that a class method in `Localeapp` (like `Localeapp.load_yaml`), or even a new dedicated class/module would be better:
* I don't think it's `Localeapp::CLI::Pull` responsibility;
* I don't see an easy way to unit-test this code here.